test/integration/floating_ip_integration_test.go

```diff
@@ -265,3 +267,85 @@ func TestIntegrationFloatingIP_PrefixLength(t *testing.T) {
 	}
 
 }
+
+func TestIntegrationFloatingIP_Global(t *testing.T) {
+	integrationTest(t)
+
+	allRegions, err := getAllRegions()
+	if err != nil {
+		t.Fatalf("getAllRegions returned error %s\n", err)
+	}
+
+	if len(allRegions) <= 1 {
+		t.Skip("Skipping MultiSite test.")
+	}
+
+	createServerRequest := &cloudscale.ServerRequest{
+		Name:         serverBaseName,
+		Flavor:       "flex-2",
+		Image:        DefaultImageSlug,
+		VolumeSizeGB: 10,
+		SSHKeys: []string{
+			pubKey,
+		},
+	}
+
+	var servers []*cloudscale.Server
+	for _, region := range allRegions {
+		createServerRequest.Zone = region.Zones[0].Slug
+		server, err := client.Servers.Create(context.Background(), createServerRequest)
+		if err != nil {
+			t.Fatalf("Servers.Create returned error %s\n", err)
+		}
+		servers = append(servers, server)
+	}
+
+	for _, server := range servers {
+		waitUntil("running", server.UUID, t)
+	}
+
+	createFloatingIPRequest := &cloudscale.FloatingIPCreateRequest{
+		IPVersion:    6,
+		PrefixLength: 56,
+		Type:         "global",
+		Server:       servers[0].UUID,
+	}
+
+	floatingIP, err := client.FloatingIPs.Create(context.TODO(), createFloatingIPRequest)
+	if err != nil {
+		t.Fatalf("floatingIP.Create returned error %s\n", err)
+	}
+
+	for _, server := range append(servers, servers...) {
+		expectedServerUUID := server.UUID
+		updateRequest := &cloudscale.FloatingIPUpdateRequest{
+			Server: expectedServerUUID,
+		}
+		ip := floatingIP.IP()
+		err = client.FloatingIPs.Update(context.Background(), ip, updateRequest)
+		if err != nil {
+			t.Fatalf("floatingIP.Update returned error %s\n", err)
+		}
+
+		actualFloatingIP, err := client.FloatingIPs.Get(context.Background(), ip)
+		if err != nil {
+			t.Fatalf("floatingIP.Get returned error %s\n", err)
+		}
```

Comment:
Could you maybe additionally check for the Floating IP region (it should be null).

Reply:
Added the check. Please have a look. Unfortunately, it's not the whole `.Region` that becomes `nil` since `.Region` is not a pointer. Should we change this?

Reply:
I would probably make it possible that it can become nil, because the current implementation can be quite confusing IMO.

Reply:
Agreed. I'll look into this.

Reply:
Done.